packages/components/checkboxes/template.njk

```diff
@@ -56,6 +56,8 @@
       <input class="nhsuk-checkboxes__input" id="{{ id }}" name="{{ name }}" type="checkbox" value="{{ item.value }}"
```

Comment:
HTML validator is complaining that the `input is missing required attribute role.`

The HTML validation seems to pass without errors on the GOV.UK example.

Reply:
As discussed on Slack, this is a known issue at the moment and will go away if all the proposed drafts and changes to the aria spec land. It's not an issue introduced by this pull request, and is present in gov-frontend also (Where they are monitoring the situation ongoing).

See also:

https://github.com/validator/validator/issues/782
https://github.com/w3c/aria/issues/1404